nautobot/extras/models/jobs.py

```diff
@@ -825,6 +860,8 @@ def to_cron(self):
             return schedules.crontab(minute=t.minute, hour=t.hour)
         elif self.interval == JobExecutionType.TYPE_WEEKLY:
             return schedules.crontab(minute=t.minute, hour=t.hour, day_of_week=t.weekday())
+        elif self.interval == JobExecutionType.TYPE_CUSTOM:
+            return self.get_crontab(self.crontab)
         raise ValueError(f"I do not know to convert {self.interval} to a Cronjob!")
```

Comment:
I think it would be good to add a housekeeping item to stash the crontab onto `self.crontab` in all cases. Could clean up a lot of these methods.